alpa/collective/collective.py

```diff
@@ -387,7 +407,8 @@ def broadcast_partialgpu(tensor_list,
                          world_size,
                          devices_ids,
                          devices_global_rank,
-                         group_name: str = "default"):
+                         group_name: str = "default",
+                         local_start_pos_list=None):
```

Comment:
Plz fix doc string as you add new arguments. The same applies for `start_pos`

Reply:
OK